app/containers/Admin/NaicsCode/NaicsCodeTable.tsx

```diff
@@ -16,15 +16,34 @@ export const NaicsCodeTableContainer: React.FunctionComponent<Props> = (
 ) => {
   const [validated, setValidated] = useState(false);
   const [showCreateModal, setShowCreateModal] = useState(false);
+  const [showActiveCodeError, setShowActiveCodeError] = useState(false);
   const {query} = props;
 
+  const handleClose = () => {
+    setShowCreateModal(false);
+    setValidated(false);
+    setShowActiveCodeError(false);
+  };
+
+  const naicsCodeIsActive = (naicsCode) => {
+    let codeExists = false;
+    query.allNaicsCodes.edges.forEach((edge) => {
+      if (edge.node.naicsCode === naicsCode) {
+        codeExists = true;
+      }
+    });
+    return codeExists;
```

Comment:
You can use `some(...)`, it returns on the first found occurrence instead of looping through the entire array

```suggestion
    return query.allNaicsCodes.edges.some((edge) => 
      edge.node.naicsCode === naicsCode
    );
```